fatips/102.md

```diff
@@ -14,93 +14,66 @@ tokens.
 
 Create a central, authoritative, public index of FAT tokens. Such an index is
 ideal for creating FAT token explorers, and improving the discoverability of
-tokens and the ecosystem.
+tokens within the ecosystem.
 
 
 # Specification
 
 
-## Index Chain
+## Token Index Chain
 
-A single chain, the index chain holds all entries supporting the standard
+The index chain holds entries corresponding to individual signed listings of FAT tokens by their issuers.
 
 
-### Index Chain ID `extends` [FATIP-100](100)
+### Token Index Chain ID `extends` [FATIP-100](100)
 
-Resource Identifier = `fat:index`
-
-```
-Chain ID = sha256(sha256(fat:index))
-Chain ID = efeb6db82bf222a87c1fc756e09197ed6ac7672641761a019c26530967f27744
-```
-
-
-## Index Entry
-
-Entries in the index chain represent individual FAT tokens being indexed.
-
-
-### Token ID `extends` [FATIP-100](100)
-
-A `tokenID` field in the index entry acts as a reference to the FAT token being
-indexed. This spec is compatible with FAT tokens implementing the
-[FATIP-100](100) Chain ID Derivation Standard.
+The first entry index chain has the following External IDs:
 
+| External ID Index | External ID Value |
+| ----------------- | ----------------- |
+| 0                 | "index"           |
+| 1                 | "tokens"          |
 
-### Salt
-
-A `salt` field in the index entry acts a source of randomness for the
-signature. It is a user defined string and is optional. Using a good salt helps
-ensures signatures are not re-playable
+For example, for FAT-0 token `mytoken`:
 
+```
+Chain ID = sha256(sha256("index") | sha256("tokens"))
+Chain ID = 9385ab32f9748bf4c24cedb301bc5a4938eb5cd4a5b0bb46b70027bf983f579a2e
+```
 
-### Digital Identity Signature `extends` [FATIP-101](101)
 
-Each entry is signed by the token issuer's digital identity. FAT tokens may
-only be publicly indexed by the issuing identity.
 
+## Token Index Entry `extends` [FATIP-101](101.md)
 
-#### Signed Data
+Entries in the index chain represent individual FAT tokens being indexed. Each entry is a simple pointer containing the necessary data to find the token's chain, where user facing data such as Name, Symbol & etc. can be found.
 
-`data` specified as per [FATIP-101](101) is derived as follows:
+From [FATIP-101](101.md), the `data` being signed shall be the Chain ID of the token chain concatenated with the index entry content. The `issuer` signing the content shall be the hex encoded Identity Root Chain ID specified in the in the entry JSON, and the `IDKey` used shall be the IDKey of `issuer` at the time of entry to Factom.
```

Comment:
'the `data` being signed shall be the Chain ID of the token chain' ==> that contradicts what is written few line below where the data being signed is defined as (`index chain ID`+ index entry content)

Reply:
Sorry that's copy pasta :spaghetti: The latter is correct (index chain ID + content)

Reply:
Resolved in commit [142f059](https://github.com/Factom-Asset-Tokens/FAT/pull/23/commits/142f059e100ed6a6e33674e6d91a108a0a45a9ea)